package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "@hyperlane-xyz/explorer",
   "description": "An interchain explorer for the Hyperlane protocol and network.",
-  "version": "3.8.0",
+  "version": "3.11.0",
   "author": "J M Rossy",
   "dependencies": {
     "@headlessui/react": "^1.7.17",
-    "@hyperlane-xyz/sdk": "3.8.0",
-    "@hyperlane-xyz/utils": "3.8.0",
+    "@hyperlane-xyz/registry": "^1.1.0",
+    "@hyperlane-xyz/sdk": "3.11.1",
+    "@hyperlane-xyz/utils": "3.11.1",
     "@hyperlane-xyz/widgets": "3.8.0",
```

Comment:
this doesnt use your widgets changes? seems like an opportunity to dedupe?

Reply:
It will now! Just needed to merge and publish that part first